observability/logging/logging-6.0/log6x-about.adoc

```diff
@@ -0,0 +1,50 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="log6x-about"]
+= About logging 6.0
+:context: logging-6x
+
+toc::[]
+
+The `ClusterLogForwarder` custom resource is the central configuration point for OpenShift Logging 6.0. It defines the desired state of your logging system, including inputs, outputs, and pipelines. The Cluster Logging Operator watches for changes to this resource and automatically applies the necessary configurations to the underlying logging components.
+
+== Service account
+This release of Cluster Logging requires administrators to explicitly grant log collection permissions to the service account associated with *ClusterLogForwarder*. This was not required in previous releases for the legacy logging scenario consisting of a *ClusterLogging* and, optionally, a *ClusterLogForwarder.logging.openshift.io* resource.
```

Comment:
This is missing if it's created a clusterrole, to indicate what are the clusterrolebinding to be able to collect audit logs, application logs and infrastructure logs as indicated in https://docs.openshift.com/container-platform/4.16/observability/logging/log_collection_forwarding/log-forwarding.html#log-collection-rbac-permissions_log-forwarding in case that not using the serviceAccount logcollector available in the ns openshift-logging